ls1intum/Artemis
# `General`: Make Exercise View Scrollable

### Checklist
#### General




#### Client


### Motivation and Context


Fixes #8674 

The Course Overview page can't be properly displayed on some mobile devices with narrow screens (even with the sidebar collapsed). This PR implements a quick fix to make the view scrollable for those screens

### Description


Added CSS to make it scrollable

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student
- 1 Programming Exercise

1. Log in to Artemis
2. Navigate to Course Overview
3. Take a look at Exercise Page with a narrow screen
4. Verify that everything is able to be displayed now


### Testserver States
> [!NOTE]
> These badges show the state of the test servers.
> Green = Currently available, Red = Currently locked

![](https://byob.yarr.is/ls1intum/Artemis/artemis-test1)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test2)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test3)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test4)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test5)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test6)

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests
#### Exam Mode Test

### Test Coverage







### Screenshots



https://github.com/ls1intum/Artemis/assets/98264403/efee73e5-a49b-447f-a5eb-f27cc026394c
